files/ie10-viewport/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "Twitter"
```

Comment:
Sure it is. It's a copy of the ie10-viewport Workaround script that is published with Twitter Bootstrap. Even the repo says the license is from Twitter Bootstrap.

https://github.com/thekondrashov/ie10-viewport